src/logic/safe/store/reducer/gatewayTransactions.ts

```diff
@@ -77,6 +78,16 @@ export const gatewayTransactionsReducer = handleActions<GatewayTransactionsState
             // pushing a newer transaction to the existing list messes the transactions order
             // this happens when most recent transactions are added to the existing txs in the store
             newHistory[startOfDate] = newHistory[startOfDate].sort((a, b) => b.timestamp - a.timestamp)
+
+            // remove tx from queued if it exists there
+            if (isMultisigExecutionInfo(transaction.executionInfo)) {
+              console.log(transaction.executionInfo.nonce)
+              const txNonce = transaction.executionInfo.nonce
```

Comment:
I think it's better to use the `safeTxHash` because module interactions don't have a nonce.